docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,51 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[run-against-an-existing-machine]]
+== Run against an Existing Machine
+
+{project} can be run against an existing machine using `vm-driver` as `generic` along with the required options.
```

Comment:
What are the required options? Are we referring to the user's _desired_ options?